### PR TITLE
Feature/solana gatekeeper cli

### PR DESCRIPTION
**Pull request by dankelleher**
Remove oclif from solana gatekeeper-lib to reduce dependency size - created a new package: gatekeeper-cli with all the oclif code.

### CODE REVIEW

File: solana/gatekeeper-cli/src/util/flags.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
-import { Keypair, PublicKey } from "@solana/web3.js";
+import {Keypair, PublicKey} from "@solana/web3.js";
 import { Flags } from "@oclif/core";
-import { readKey } from "../account";
-import { ExtendedCluster } from "../connection";
+import * as fs from "fs";
+import { ExtendedCluster } from "./utils";
+// import { ExtendedCluster } from "@identity.com/solana-gatekeeper-lib";
```
Comment (JulianSpring) [inline solana/gatekeeper-cli/src/util/flags.ts:5]: Can remove

File: solana/gatekeeper-cli/src/util/utils.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
-import { Connection, Keypair, PublicKey } from "@solana/web3.js";
-import { ExtendedCluster, getClusterUrl, getConnection } from "../connection";
-import { airdropTo } from "../account";
-import { GatekeeperService } from "../../service";
+import {Cluster, clusterApiUrl, Connection, Keypair, PublicKey} from "@solana/web3.js";
+import {airdropTo, GatekeeperService, getConnection} from "@identity.com/solana-gatekeeper-lib";
+import { GatewayToken } from "@identity.com/solana-gateway-ts";
+// import {CIVICNET_URL} from "@identity.com/solana-gatekeeper-lib";
```
Comment (JulianSpring) [inline solana/gatekeeper-cli/src/util/utils.ts:4]: Cam remove

File: solana/gateway-ts/docs/classes/Active.html [hunk 3]
```diff
@@ -111,7 +111,7 @@ <h3>constructor</h3>
 							<aside class="tsd-sources">
 								<p>Inherited from Assignable.constructor</p>
 								<ul>
-									<li>Defined in <a href="https://github.com/identity-com/on-chain-identity-gateway/blob/83df0519/solana/gateway-ts/src/lib/solanaBorsh.ts#L7">lib/solanaBorsh.ts:7</a></li>
+									<li>Defined in <a href="https://github.com/identity-com/on-chain-identity-gateway/blob/17c33ed6/solana/gateway-ts/src/lib/solanaBorsh.ts#L7">lib/solanaBorsh.ts:7</a></li>
```
Comment (JulianSpring) [inline solana/gateway-ts/docs/classes/Active.html:114]: Are all the change to the docs html generated?

**Review by JulianSpring — APPROVED** [review]
Small comments

File: solana/gatekeeper-cli/src/util/flags.ts (hunk 1 above)
Comment (dankelleher) [reply solana/gatekeeper-cli/src/util/flags.ts:5]: done

File: solana/gatekeeper-cli/src/util/utils.ts (hunk 2 above)
Comment (dankelleher) [reply solana/gatekeeper-cli/src/util/utils.ts:4]: done

File: solana/gateway-ts/docs/classes/Active.html (hunk 3 above)
Comment (dankelleher) [reply solana/gateway-ts/docs/classes/Active.html:114]: Yes via oclif

**Pull request merged**